scripts/utils/create_package_template.py

```diff
@@ -436,30 +445,55 @@ def main(argv=None):
     epilog = textwrap.dedent(
         """
     Example usage:
-      python %(prog)s --pkg_name <> --version <> --authors <> --tool_name <> --category <>
+      Show available package template types
+       %(prog)s --type
+
+      Show required arguments for tool distributed in a ZIP file (ZIP_EXE template)
+       %(prog)s --type ZIP_EXE
+
+      Create package template for tool distributed in a ZIP file
+       %(prog)s --type ZIP_EXE --pkg_name new-tool --version 1.0.0 --authors 'Alice, Bob' ...
+
+      Create package template files for a METAPACKAGE using placeholder data
+       %(prog)s --type METAPACKAGE --raw
```

Comment:
This creates:
```
packages/pkg-name-CHANGEME.vm/tools
packages/pkg-name-CHANGEME.vm/pkg-name-CHANGEME.vm.nuspec 
packages/pkg-name-CHANGEME.vm/tools/chocolateyinstall.ps1
packages/pkg-name-CHANGEME.vm/tools/chocolateyuninstall.ps1
```

with for example `packages/pkg-name-CHANGEME.vm/pkg-name-CHANGEME.vm.nuspec`
```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
  <metadata>
    <id>pkg-name-CHANGEME.vm</id>
    <version>version-CHANGEME</version>
    <authors>authors-CHANGEME</authors>
    <description>description-CHANGEME</description>
    <dependencies>
      <dependency id="common.vm" />
      <dependency id="dependency-CHANGEME" version="[version-CHANGEME]" />
    </dependencies>
  </metadata>
</package>
```